src/backend/tests/risks.test.ts

```diff
@@ -144,5 +144,33 @@ describe('Risks', () => {
       expect(prisma.risk.findUnique).toHaveBeenCalledTimes(1);
       expect(prisma.risk.update).toHaveBeenCalledTimes(0);
     });
+
+    test('the user does not have permissions to create the risk', async () => {
+      jest.spyOn(prisma.project, 'findUnique').mockResolvedValue(null);
+      jest.spyOn(prisma.risk, 'create').mockResolvedValue(prismaRisk2);
+      jest.spyOn(prisma.risk, 'update').mockResolvedValue(prismaRisk2);
+      jest.spyOn(riskUtils, 'hasRiskPermissions').mockResolvedValue(true);
+
+      const projectId = 1;
+      const detail = 'detail';
+      await expect(() => RisksService.createRisk(wonderwoman, projectId, detail)).rejects.toThrow();
```

Comment:
should specify what error is being thrown

Reply:
Updated this